// @expo-google-fonts/playfair-display/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const PlayfairDisplay_400Regular = require('./400Regular/PlayfairDisplay_400Regular.ttf');
export const PlayfairDisplay_500Medium = require('./500Medium/PlayfairDisplay_500Medium.ttf');
export const PlayfairDisplay_600SemiBold = require('./600SemiBold/PlayfairDisplay_600SemiBold.ttf');
export const PlayfairDisplay_700Bold = require('./700Bold/PlayfairDisplay_700Bold.ttf');
export const PlayfairDisplay_800ExtraBold = require('./800ExtraBold/PlayfairDisplay_800ExtraBold.ttf');
export const PlayfairDisplay_900Black = require('./900Black/PlayfairDisplay_900Black.ttf');
export const PlayfairDisplay_400Regular_Italic = require('./400Regular_Italic/PlayfairDisplay_400Regular_Italic.ttf');
export const PlayfairDisplay_500Medium_Italic = require('./500Medium_Italic/PlayfairDisplay_500Medium_Italic.ttf');
export const PlayfairDisplay_600SemiBold_Italic = require('./600SemiBold_Italic/PlayfairDisplay_600SemiBold_Italic.ttf');
export const PlayfairDisplay_700Bold_Italic = require('./700Bold_Italic/PlayfairDisplay_700Bold_Italic.ttf');
export const PlayfairDisplay_800ExtraBold_Italic = require('./800ExtraBold_Italic/PlayfairDisplay_800ExtraBold_Italic.ttf');
export const PlayfairDisplay_900Black_Italic = require('./900Black_Italic/PlayfairDisplay_900Black_Italic.ttf');
